reports/usa.json

```diff
@@ -27,20 +27,6 @@
       }
     },
     {
-      "name": "last-48-hours",
```

Comment:
The `last-48-hours` report has been there for a while, and I'm not sure why it was there.